reportextractor/src/main/scala/com/gu/notifications/extractor/Lambda.scala

```diff
@@ -0,0 +1,126 @@
+package com.gu.notifications.extractor
+
+import java.time.temporal.{ChronoUnit, TemporalUnit}
+import java.time.{Duration, LocalDate}
+
+import aws.AsyncDynamo.{keyBetween, keyEquals}
+import aws.DynamoJsonConversions
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.dynamodbv2.model._
+import com.amazonaws.services.dynamodbv2.{AmazonDynamoDB, AmazonDynamoDBClientBuilder}
+import com.amazonaws.services.lambda.runtime.{Context, RequestHandler}
+import com.amazonaws.services.s3.AmazonS3ClientBuilder
+import com.gu.{AppIdentity, AwsIdentity}
+import models.NotificationType
+import org.slf4j.{Logger, LoggerFactory}
+import play.api.libs.json.{JsValue, Json}
+import utils.MobileAwsCredentialsProvider
+
```

Comment:
Think you've got a couple unused dependencies in here
AmazonDynamoDB, Duration, TemporalUnit

Reply:
Yep, good eye here!